javaparser-core/src/main/java/com/github/javaparser/ast/body/EnumDeclaration.java

```diff
@@ -140,7 +141,7 @@ public EnumDeclaration setImplementedTypes(NodeList<ClassOrInterfaceType> implem
     }
 
     public EnumConstantDeclaration addEnumConstant(String name) {
-        EnumConstantDeclaration enumConstant = new EnumConstantDeclaration(assertNotNull(name));
+        EnumConstantDeclaration enumConstant = new EnumConstantDeclaration(name);
```

Comment:
As most expressions, statements and declarations have a SimpleName and all string aka (name, field, identifier) are passed to a new SimpleName instance many checks were not necessary now.